### PR TITLE
TF-TRT enable BiasAdd op in dynamic shape mode

### PR DESCRIPTION
**Pull request by tfeher**
Improve the converter of BiasAdd to handle explicit batch mode, and test it both in explicit batch and dynamic shape mode. 

Tagging @bixia1 for review.

### CODE REVIEW

**Comment by gbaned** [comment]
@tfeher  Can you please resolve conflicts? Thanks!

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc [hunk 1]
```diff
@@ -404,6 +404,9 @@ Status GetTrtBroadcastShape(const TRT_TensorOrWeights& operand_l,
   // Compare broadcast feasibility
   if (check_feasibility) {
     for (int i = 0; i < broadcast_num_dims; ++i) {
+      if (!use_implicit_batch && (output_l[i] == -1 || output_r[i] == -1)) {
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:407]: Can we add comment here to explain the decision here? In particular, why we allow cases like these:
   (output_l[i], output_r[i]) = (-1, -1), (40, -1)
With implicit mode, we don't have (-1, -1), we only support (-1, 1) or (1, -1).

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc [hunk 2]
```diff
@@ -3891,8 +3894,20 @@ Status ConvertBiasAdd(OpConverterParams* params) {
   // If the input is NCHW, then we need to unsqueeze the bias such that its last
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:3894]: The existing comment is only for the implicit batch mode. Let's update the comment to include explicit batch mode.

Comment (tfeher) [reply tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc]: Updated it.

**Review by tfeher — COMMENTED** [review]
Thanks @bixia1 for the review, I have addressed the issues. Let me know if there are other places which needs improvements.

**Review by bixia1 — APPROVED** [review]

**Pull request merged**